packages/jsii-pacmak/lib/targets/dotnet.ts

```diff
@@ -124,19 +124,19 @@ export class DotnetBuilder implements TargetBuilder {
 
     // If dotnet-jsonmodel is checked-out and we can find a local repository, add it to the list.
     try {
-      /* eslint-disable @typescript-eslint/no-var-requires */
+      /* eslint-disable @typescript-eslint/no-var-requires,import/no-extraneous-dependencies */
```

Comment:
is this intentional? why not model it as a dep? what happens at runtime? probably worth a comment explaining why this is ok

Reply:
The comment is literally on line 125... This is a "shadow-dep" so this can work with a local-build when working within the mono-repo. If we modeled that dependency, we'd be modeling a dependency on a _private_ package.

On the worst case, this would be a `devDependency`, which would still require explicit opt-in!